test/dotnet-test-nunit.test/project.json

```diff
@@ -10,32 +10,23 @@
   },
 
   "dependencies": {
-    "NUnit": "3.2.0",
-    "NUnit.Portable.Agent": "3.4.0-alpha-2",
+    "NUnit": "3.4.0",
     "dotnet-test-nunit": {
       "target": "project"
     }
   },
   "testRunner": "nunit",
 
   "frameworks": {
-    "netstandard1.5": {
-      "imports": [
-        "dnxcore50",
-        "netstandard1.5",
-        "portable-net45+win8"
-      ],
+    "netcoreapp1.0": {
+      "imports": "dnxcore50",
```

Comment:
`dnxcore50` is not officially depricated moniker. Reference: https://github.com/dotnet/corefx/blob/96fd6fdbae3ea8c0419999d89908dfd28bcb8136/Documentation/architecture/net-platform-standard.md#deprecated-monikers (the VS2015Update3 project template for netcore console and lib haven't been updated yet accordingly, but this doc is genuine).

You may want to try `"imports": ["dotnet5.6"]` or `"imports": ["portable-net45+win8"]` or (if no individaul works; combine) `"imports": ["dotnet5.6", "portable-net45+win8"]`.